tests/e2e/package.json

```diff
@@ -42,5 +42,8 @@
     "@eclipse-che/api": "7.5.0-SNAPSHOT",
     "inversify": "5.0.1",
     "reflect-metadata": "0.1.13"
+  },
+  "resolutions": {
+    "minimist": "^1.2.5"
```

Comment:
I am not familiar with this tool but I suppose that using of it also requires a CQ

Reply:
it's already included in some of our 'che' dependencies

https://github.com/eclipse/che-workspace-loader/blob/master/yarn.lock#L4261